lib/DocumentManager.php

```diff
@@ -175,4 +265,34 @@ private function getOptionsResolver($eventName)
 
         return $resolver;
     }
+
+    /**
+     * If the exception is an instanceof `DocumentManagerException` set the
+     * document manager name that threw.
+     *
+     * @param \Exception $e
+     * @param string $context
+     */
+    private function processException(\Exception $exception, $context)
```

Comment:
I don't really care. The point is that we repeat the same logic ~10
times in this class (catching the Exception) so anything we can do to
minimize the responsibility of each of those try/catch blocks the
better.

If you really feel strongly about this than it can be changed.

On Thu, Apr 28, 2016 at 12:44:58AM -0700, Daniel Rotter wrote:
>    In [1]lib/DocumentManager.php:
> 
>  > @@ -175,4 +265,34 @@ private function getOptionsResolver($eventName)
>  >
>  >          return $resolver;
>  >      }
>  > +
>  > +    /**
>  > +     * If the exception is an instanceof `DocumentManagerException` set the
>  > +     * document manager name that threw.
>  > +     *
>  > +     * @param \Exception $e
>  > +     * @param string $context
>  > +     */
>  > +    private function processException(\Exception $exception, $context)
> 
>    What kind of things? The DocumentManager itself should only handle
>    exceptions of type DocumentManagerException IMO anyway, so there is no
>    need to change the method signature later.
> 
>    —
>    You are receiving this because you authored the thread.
>    Reply to this email directly or [2]view it on GitHub
> 
>    Reverse link: [3]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/sulu/sulu-document-manager/pull/66#discussion_r61385314
>    2. https://github.com/sulu/sulu-document-manager/pull/66/files/05556aabdb564b020fa388baf338b54d39353978#r61385314
>    3. https://github.com/sulu/sulu-document-manager/pull/66/files/05556aabdb564b020fa388baf338b54d39353978#r61385314